main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -89,9 +89,10 @@ object ParsedAst {
       * @param tparams    the type parameters.
       * @param fparamsOpt the formal parameters.
       * @param tpe        the declared type.
+      * @param exp        the expression.
```

Comment:
The optional expression.

Reply:
A signature now carries an option expression. I think this makes sense, but I wonder if we should start thinking about another name for "signatures" / "class members"?

Reply:
Yes, I think we still have a lot of terminology questions. Probably worth opening a ticket.